### ISSUE TITLE
Change or allow changing location of config file

### ISSUE THREAD

**Issue by jaworek** [andreasgrill/auto-selfcontrol#58]
Having config file in HOME directory would be useful for backup and sharing.
Would it be possible to change it to follow XDG Base Directory Specification?
Is there a reason for config file to be in the current directory?

I'm more than happy to create PR for that, although it seems that it will be required to create a migration script for current users.

https://wiki.archlinux.org/index.php/XDG_Base_Directory

**Comment by sbibauw** [comment]
I agree with @jaworek. Plus the current workflow does not work on new M1 Macs (where Homebrew installs at `/opt/homebrew/`. 

I created PR #62 to solve this. It doesn't do the migration, but it will not break for previous config locations.

**Comment by Pandoks** [comment]
I created [Hammer Control](https://github.com/Pandoks/hammer-control) which inputs the password into the prompt of the newest version of Self Control. It also lets you define a specific blacklist for every scheduled session which seems like something that you'd like.
